contrib/platform/test/com/sun/jna/platform/win32/Kernel32UtilTest.java

```diff
@@ -253,4 +256,34 @@ public final void testWritePrivateProfileSection() throws IOException {
             reader.close();
         }        
     }
+    
+    public void testGetResource() {
+		// On Windows 7, "14" is the type assigned to the "My Computer" icon
+		// (which is named "ICO_MYCOMPUTER")
+    	byte[] results = Kernel32Util.getResource("c:\\windows\\explorer.exe", "14", "ICO_MYCOMPUTER");
```

Comment:
Should I have used "%WINDIR%\\explorer.exe" instead?

If not can you provide some info on your unit test boxes (eg Windows version, etc)?

I was hoping to use something that already existed rather than have to upload a binary blob (and the Visual Studio project for it too presumably). 

Sent from my iPhone

> On Nov 11, 2015, at 6:55 AM, Daniel Doubrovkine (dB.) @dblockdotorg <notifications@github.com> wrote:
> 
> In contrib/platform/test/com/sun/jna/platform/win32/Kernel32UtilTest.java:
> 
> > @@ -253,4 +256,34 @@ public final void testWritePrivateProfileSection() throws IOException {
> >              reader.close();
> >          }        
> >      }
> > +    
> > +    public void testGetResource() {
> > +		// On Windows 7, "14" is the type assigned to the "My Computer" icon
> > +		// (which is named "ICO_MYCOMPUTER")
> > +    	byte[] results = Kernel32Util.getResource("c:\\windows\\explorer.exe", "14", "ICO_MYCOMPUTER");
> This doesn't exist on everybody's computer. Not everyone has a C drive. Really.
> 
> —
> Reply to this email directly or view it on GitHub.
>